app/views/catalog/availability/_badges.html.erb

```diff
@@ -1,4 +1,6 @@
-<% if doc_avail_values.present? %>
+<% holdings_available = doc_avail_values.present? && (doc_avail_values[:online_available] || document.url_fulltext.present? || doc_avail_values[:physical_holdings].any?) %>
```

Comment:
I am assuming a document is available when the following is true:
- We must receive availability data from the ALMA API
- The document either has physical holdings, is available online, or it has a full text url we can link to